website/src/index.tsx

```diff
@@ -5,14 +5,14 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom'
 import App from './components/App'
 import Login from './components/views/Admin/Login'
 import Signup from './components/views/Admin/Signup'
-
+import Header from './components/views/User/Header'
 ReactDOM.render(
 
   <BrowserRouter>
     <Switch>
-      <Route exact path='/' render={()=><App/>}/>
+      <Route exact path='/' render={()=><Header ><App/></Header>}/>
```

Comment:
This is not a good way to render components in a route. You can either put the `Header `component inside the `App` component and supply the App component as a **child** to the `Route`.